font-codegen/src/record.rs

```diff
@@ -40,6 +40,11 @@ pub(crate) fn generate(item: &Record) -> syn::Result<TokenStream> {
             impl FixedSize for #name {
                 const RAW_BYTE_LEN: usize = #( #inner_types::RAW_BYTE_LEN )+*;
             }
+
+            /// SAFETY: I am repr(c) + repr(packed), and all of my fields impl `JustBytes`.
+            unsafe impl JustBytes for #name {
```

Comment:
Optional, how do you feel about FromBytes or some other name that more clearly indicates it's safe to convert this from raw bytes?